static/templates/compose_control_buttons.hbs

```diff
@@ -1,8 +1,8 @@
 <div class="compose_control_buttons_container order-1">
     <a role="button" class="markdown_preview compose_control_button fa fa-eye" aria-label="{{t 'Preview' }}" tabindex=0 data-tippy-content="{{t 'Preview' }}"></a>
     <a role="button" class="undo_markdown_preview compose_control_button fa fa-edit" aria-label="{{t 'Write' }}" tabindex=0 style="display:none;" data-tippy-content="{{t 'Write' }}"></a>
-    <div class="divider">|</div>
-    <div class="compose_control_buttons_container hiddable_compose_control_buttons">
+    <div class="compose_control_buttons_container order-1 preview_mode_hidden">
+        <div class="divider">|</div>
```

Comment:
We don't need `compose_control_buttons_container order-1 ` here, right?

Reply:
It is for the CSS on a tags.

Reply:
![removed_compose_buttons_container_class](https://user-images.githubusercontent.com/41695888/152821326-3b86f39a-70b1-41cd-a93a-e2496029d683.png)

Reply:
Ok, cool!